go.mod

```diff
@@ -27,10 +27,15 @@ require (
 	go.etcd.io/bbolt v1.3.7
 )
 
+replace decred.org/dcrwallet/v4 => ../dcrwallet
```

Comment:
This shouldn't be committed as it doesn't work for consumers.  If you want it for local purposes, you should put it in your local `go.work`.